plugin.py

```diff
@@ -55,7 +55,8 @@
                 <option label="MQTT discovery" value="MQTTDiscovery"/>
                 <option label="Onkyo AV Receiver" value="Onkyo"/>
                 <option label="OpenAQ" value="xfr_openaq"/>
-                <option label="Pi-hole summary" value="xfr_pihole"/>
```

Comment:
Your "commit" says that "Pi_hole summary" will be removed.  Sorry, cannot merge it like that.
Please correct your code or delete your request and repost it!

regards,